evap/student/templates/student_vote.html

```diff
@@ -132,6 +132,30 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
             // (the data gets deleted every time the form is submitted, i.e. also when the form had errors and is displayed again)
             sisyphus.saveAllData();
 
+            // Show a warning on textareas if they only contain obsolete text (punctuation, k.A., n/a, ...)
+            function isTextareaInvalid(textarea) {
+                let val = textarea.val();
+                return /^\W+$/g.test(val) || ["ka", "na"].indexOf(val.toLowerCase().replace(/\W/g,'')) != -1;
+            }
+            function updateTextareaWarning(textarea) {
+                let showWarning = isTextareaInvalid(textarea);
+                textarea.parents('tr').find('.warning-label').toggleClass('d-none', !showWarning);
```

Comment:
i have the feeling `.warning-label` is a weird selector here, because that's a styling thing and not some identification thing, but i don't really know what i'm talking about, @janno42?

Reply:
that's fine for me.